SPECS/gd/gd.spec

```diff
@@ -0,0 +1,636 @@
+%define prjname libgd
+%define lname libgd3
+Summary:        A Drawing Library for Programs That Use PNG and JPEG Output
+Name:           gd
+Version:        2.3.0
+Release:        2.3%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://libgd.github.io/
+Source0:        https://github.com/libgd/libgd/releases/download/%{name}-%{version}/%{prjname}-%{version}.tar.xz
+Source1:        baselibs.conf
+# might be upstreamed, but could be suse specific also (/usr/share/fonts/Type1 font dir)
+Patch1:         gd-fontpath.patch
+# could be upstreamed, but not in this form (need ac check for attribute format printf, etc.)
+Patch2:         gd-format.patch
+# could be upstreamed
+Patch3:         gd-aliasing.patch
+BuildRequires:  fontconfig-devel
+BuildRequires:  freetype > 2
+# needed for tests
```

Comment:
Please surround any test-only dependencies with `%if %{with_check}`.

Reply:
Test code is built by default. So these build requires are required by default. I'll remove the comments and run the test inside macro.

Reply:
If by "Test code is built by default" you mean that `%make_build` from the `%build` section (**NOT** the `%check` section) fails without these BRs, then I agree with your suggestion. If it passes, however, then we would still use the `%if` here.